.github/workflows/scripts/e2e-verify.common.sh

```diff
@@ -56,3 +56,45 @@ e2e_verify_common_metadata() {
 e2e_verify_common_materials() {
     e2e_verify_predicate_materials "$1" "{\"uri\":\"git+https://github.com/$GITHUB_REPOSITORY@$GITHUB_REF\",\"digest\":{\"sha1\":\"$GITHUB_SHA\"}}"
 }
+
+# Verifies common fields of the SLSA v1.0 predicate.
```

Comment:
Did we copy this file from example-package at some point? Any thoughts on how we could manage drift between the tests?